test/run-e2e-ocp.sh

```diff
@@ -22,25 +22,26 @@ declare SHOW_USAGE=false
 
 cleanup() {
 	# NOTE: clears trap INT
-	trap - INT
+	info "Cleaning up ocp..."
```

Comment:
header instead of "info" :)

Reply:
`cleanup()` of `run-e2e.sh` has `info`, suppose here should be consistent